packages/smooth_app/lib/knowledge_panel/knowledge_panels/knowledge_panel_page.dart

```diff
@@ -42,6 +42,7 @@ class _KnowledgePanelPageState extends State<KnowledgePanelPage>
   @override
   Widget build(BuildContext context) {
     return SmoothScaffold(
+      brightness: Brightness.dark,
```

Comment:
This page is used both for onboarding and from the "normal" product page: should we set the brightness to dark in all cases?

Reply:
No you're, right, I will add an argument to override only in the walkthrough